### ISSUE TITLE
Alt attribute

### ISSUE THREAD

**Issue by jim-rosales** [cpasquinelli/prj-rev-bwfs-dasmoto#6]
Usually, it is best practice to include the [alt attribute](https://en.wikipedia.org/wiki/Alt_attribute) for images used on your site. The reasons for this are because the attribute is used for [accessibility](https://www.codecademy.com/articles/accessibility) and as a backup, in case your image cannot be shown. 

For example:

`<img src="https://s3.amazonaws.com/codecademy-content/courses/freelance-1/unit-2/hacksaw.jpeg" alt="Hacksaw" />`

https://github.com/cpasquinelli/prj-rev-bwfs-dasmoto/blob/ddd5736f9aced1707fb245475e20dc69cafb4357/Arts%26Crafts/index.html#L22
